### DISCUSSION TITLE
New release 1.6.3

### DISCUSSION THREAD

**Discussion by pablosnt** [pablosnt/rekono#242] [Announcements]
### Fixed

- Upgrade `pyyaml` version to `6.0.1` (https://github.com/pablosnt/rekono/issues/240)

<hr /><em>This discussion was created from the release <a href='https://github.com/pablosnt/rekono/releases/tag/1.6.3'>1.6.3</a>.</em>

**Comment by zinwelzl** [comment]
This error after docker-compose build

+++++++++++++++++++++++++++++++++++++++++++++++++++
nginx                | req: Can't open "/etc/nginx/tls/privatekey.key" for writing, Permission denied
nginx exited with code 1
backend              | 2023-07-26 13:20:01,611 [INFO] - 17 middleware - 127.0.0.1 - anonymous - GET /api/schema/swagger-ui.html > HTTP 200                                  
frontend             |  HTTP  7/26/2023 11:20:09 AM 127.0.0.1 GET /
frontend             |  HTTP  7/26/2023 11:20:09 AM 127.0.0.1 Returned 200 in 3 ms
backend              | 2023-07-26 13:20:11,725 [INFO] - 17 middleware - 127.0.0.1 - anonymous - GET /api/schema/swagger-ui.html > HTTP 200
frontend             |  HTTP  7/26/2023 11:20:19 AM 127.0.0.1 GET /
frontend             |  HTTP  7/26/2023 11:20:19 AM 127.0.0.1 Returned 200 in 3 ms
backend              | 2023-07-26 13:20:21,863 [INFO] - 17 middleware - 127.0.0.1 - anonymous - GET /api/schema/swagger-ui.html > HTTP 200
^CGracefully stopping... (press Ctrl+C again to force)                                                                                                                      
Stopping nginx                      ... done
Stopping rekono_executions-worker_1 ... done
Stopping telegram-bot               ... done
Stopping backend                    ... done
Stopping rekono_emails-worker_1     ... done
Stopping rekono_tasks-worker_1      ... done
Stopping rekono_findings-worker_1   ... done
Stopping redis                      ... done
Stopping frontend                   ... done
Stopping postgres                   ... done

**Reply by pablosnt** [reply]
Yeah, this is a common error in some environments. Check the solution in the [FAQ](https://github.com/pablosnt/rekono/wiki/FAQ#permission-error-during-nginx-execution)
